HTTP UA Client Hint headers (#11191)
mdn/content: files/en-us/web/http/headers/sec-ch-ua-model/index.md

tags:
  - Sec-CH-UA-Model
  - Client hint
  - HTTP
  - HTTP Header
  - Reference
  - Request header
  - Exerimental
browser-compat: http.headers.Sec-CH-UA-Model
---
{{HTTPSidebar}} {{SeeCompatTable}} {{securecontext_header}}

The **`Sec-CH-UA-Model`** [user agent client hint](/en-US/docs/Glossary/Client_hints#user-agent_client_hints) request header indicates the device model on which the browser is running.

<table class="properties">
  <tbody>
    <tr>
      <th scope="row">Header type</th>
      <td>
        {{Glossary("Request header")}},
        {{Glossary("Client hints","Client hint")}}
      </td>
    </tr>
    <tr>
      <th scope="row">{{Glossary("Forbidden header name")}}</th>
      <td>yes</td>
    </tr>
  </tbody>
</table>

## Syntax

```http
Sec-CH-UA-Model: <device-version>
```

### Directives

- `<device-version>`
  - : A string containing the device version. For example "Pixel 3".

## Examples

A server requests the `Sec-CH-UA-Model` header by including the {{HTTPHeader("Accept-CH")}} in a _response_ to any request from the client, using the name of the desired header as a token:

```http
HTTP/1.1 200 OK
Accept-CH: Sec-CH-UA-Model
```

> The client may then opt-in to providing the hint, and add the `Sec-CH-UA-Model` header to subsequent requests.

Author: ```suggestion
The client may choose to provide the hint, and add the `Sec-CH-UA-Model` header to subsequent requests.
```
